editor/example/json-spec/spatial-layout.ts

```diff
@@ -133,20 +123,22 @@ export const EX_SPEC_SPATIAL_MULTIPLE_TRACKS: GoslingSpec = {
     title: 'Spatial Layout: multiple tracks in one view',
     views: [
         {
-            layout: 'spatial',
+            layout: {
+                type: 'spatial',
+                model: {
+                    type: 'csv',
+                    url: 'https://raw.githubusercontent.com/dvdkouril/chromospace-sample-data/main/gosling-3d/yeast_model.csv',
+                    xyz: ['x', 'y', 'z'],
+                    chromosome: 'chr',
+                    position: 'coord'
+                }
+            },
             tracks: [
                 {
                     data: {
                         type: 'csv',
                         url: 'https://raw.githubusercontent.com/dvdkouril/chromospace-sample-data/main/gosling-3d/yeast_model.csv'
                     },
-                    spatial: {
-                        x: 'x',
-                        y: 'y',
-                        z: 'z',
-                        chr: 'chr',
-                        coord: 'coord'
-                    },
```

Comment:
The changes I made in examples are basically removing `"spatial"` and adding `"layout: { type: 'spatial', model: { ...}, ... }`. But, using the previous approach, i.e., "spatial", should also work as well (i.e., Gosling will render the same thing).